recipes/r-rjags/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set version = '4-6' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rjags
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rjags_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rjags_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rjags/rjags_{{ version }}.tar.gz
+  patches:
+    - fix-module-path.diff
+
+  sha256: cf24bb1e7c8445bafb49097089ad33e5bd5d8efbccf16fc7e32ad230f05f89ad
+
+build:
+  number: 0
+  skip: true  # [win]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+    - lib/JAGS/modules-4/
+
+requirements:
+  build:
+    - r-base
+    - r-coda >=0.13
+    - gcc
+    - jags 4.*
+
+  run:
+    - r-base
+    - r-coda >=0.13
+    - libgcc
+    - jags 4.*
+
+test:
+  commands:
+    - $R -e "library('rjags')"
+
+about:
+  home: http://mcmc-jags.sourceforge.net
+  license: GPL (== 2)
+  summary: Interface to the JAGS MCMC library.
+  license_family: GPL2
+
+extra:
+  recipe-maintainers:
+    - micknudsen
```

Comment:
If you like, please include a few more maintainers: https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml

Reply:
Should I just add everybody on the list?

Reply:
Yes, please.